src/character.cpp

```diff
@@ -596,6 +596,54 @@ std::vector<item_location> Character::nearby( const std::function<bool(const ite
     return res;
 }
 
+long int Character::i_add_to_container(const item &it)
```

Comment:
Switched to visit_items. Won't let me use visit_items in Pickup namespace. Would have to change it to a class and inherit like in character.h?

```Character::Character() : Creature(), visitable<Character>()```